core/meterer/meterer.go

```diff
@@ -98,6 +99,7 @@ func (m *Meterer) MeterRequest(ctx context.Context, header core.PaymentMetadata,
 
 // ServeReservationRequest handles the rate limiting logic for incoming requests
 func (m *Meterer) ServeReservationRequest(ctx context.Context, header core.PaymentMetadata, reservation *core.ReservedPayment, numSymbols uint, quorumNumbers []uint8) error {
+	m.logger.Info("Recording and validating reservation usage", "reservation", reservation)
```

Comment:
Should we add header to this log and below one too?

Reply:
yeah, added:)